src/compas/robots/resources/basic.py

```diff
@@ -82,7 +82,7 @@ def can_load_mesh(self, url):
             Otherwise ``False``.
         """
 
-        url = self._get_local_path(url)
+        url = self._get_local_url(url)
```

Comment:
I think the suggested `_get_mesh_url` is more appropriate, because it does not return only local urls, it can be remote as well.

Reply:
Sorry, I overlooked the suggestion was `_get_mesh_url` instead of `_get_local_url`. I will fix that.